core/src/main/java/org/pentaho/di/core/util/StringEvaluator.java

```diff
@@ -255,6 +255,7 @@ private void evaluateLength( String value ) {
     }
   }
 
+
```

Comment:
Why are you updating this file just to add a blank line?
Please revert this file.